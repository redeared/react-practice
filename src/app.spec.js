import React from 'react';
import renderer from 'react-test-renderer';
import { App } from './app';

describe('App', () => {
    it('renders without crashing', () => {
        const component = renderer.create(<App />).toJSON();
        expect(component).toMatchSnapshot();
    });
});
