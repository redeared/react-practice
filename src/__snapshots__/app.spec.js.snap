// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders without crashing 1`] = `
<div
  style={
    Object {
      "backgroundImage": "url('https://assets.nflxext.com/ffe/siteui/vlv3/80ffdc60-07ad-4e9e-829f-7969df93981c/524ebc0c-5887-4814-9015-7551eeb56858/RU-en-20180423-popsignuptwoweeks-perspective_alpha_website_small.jpg')",
      "bottom": 0,
      "display": "flex",
      "flexDirection": "column",
      "height": "100%",
      "left": 0,
      "margin": "0 2em",
      "overflow": "auto",
      "position": "fixed",
      "right": 0,
      "top": 0,
    }
  }
>
  <div
    style={
      Object {
        "flexBasis": 0,
      }
    }
  >
    <div
      style={
        Object {
          "background": "rgba(0, 0, 0, 0.7)",
          "color": "white",
        }
      }
    >
      <span
        style={
          Object {
            "color": "#f45362",
            "fontWeight": "bold",
            "paddingLeft": "4em",
          }
        }
      >
         netflixroulette 
      </span>
    </div>
  </div>
  <div
    style={
      Object {
        "display": "flex",
        "flexDirection": "column",
        "height": "100%",
        "overflowY": "auto",
      }
    }
  >
    <div
      style={
        Object {
          "background": "rgba(0, 0, 0, 0.7)",
          "color": "white",
          "paddingLeft": "4em",
          "paddingRight": "4em",
        }
      }
    >
      <div>
        FIND YOUR MOVIE
      </div>
      <div>
        <input
          onChange={[Function]}
          onKeyPress={[Function]}
          type="text"
        />
      </div>
      <div>
        SEARCH BY: 
        <button
          onClick={[Function]}
          style={
            Object {
              "backgroundColor": "#f45362",
            }
          }
        >
          TITLE
        </button>
        <button
          onClick={[Function]}
          style={null}
        >
          GENRE
        </button>
        <div
          style={
            Object {
              "float": "right",
            }
          }
        >
          <button
            onClick={[Function]}
          >
            SEARCH
          </button>
        </div>
      </div>
    </div>
    <div
      style={
        Object {
          "alignItems": "center",
          "background": "white",
          "display": "flex",
          "flexDirection": "column",
          "height": "100%",
          "justifyContent": "center",
        }
      }
    >
      <h1>
         No films found 
      </h1>
    </div>
  </div>
  <div
    style={
      Object {
        "flexBasis": 0,
      }
    }
  >
    <div
      style={
        Object {
          "background": "rgba(0, 0, 0, 0.7)",
          "color": "white",
        }
      }
    >
      <span
        style={
          Object {
            "color": "#f45362",
            "fontWeight": "bold",
            "paddingLeft": "4em",
          }
        }
      >
        netflixroulette
      </span>
    </div>
  </div>
</div>
`;
